app/views/Export.js

```diff
@@ -142,12 +141,12 @@ export const ExportScreen = ({ navigation }) => {
                 {t('share.paragraph_second')}
               </Typography>
 
-              <TouchableOpacity style={styles.exportButton} onPress={onShare}>
-                <Typography style={styles.exportButtonText}>
-                  {t('share.button_text')}
-                </Typography>
-                <SvgXml style={styles.exportIcon} xml={exportIcon} />
-              </TouchableOpacity>
+              <Button
```

Comment:
<img width="375" alt="Screen Shot 2020-05-04 at 9 38 30 PM" src="https://user-images.githubusercontent.com/20157849/81030773-a0d38a00-8e4f-11ea-9e9b-a94196ad1f02.png">

The only styling this button has is a `marginTop: 48`, but it looks like the padding for the label and icon is still pretty close to the edge of the button. What do you think about adding

```css
padding-left: 10;
```

to `const Label = styled(Typography)` ?